.github/workflows/PR.yml

```diff
@@ -8,7 +8,11 @@ on:
   workflow_call:
   workflow_dispatch:
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
```

Comment:
test:
```suggestion


```

Reply:
it worked:

<img width="1133" alt="image" src="https://github.com/user-attachments/assets/6f82240d-d720-486f-961d-2f06fa9b89b5">